tests/regression/tests/REQUEST-933-APPLICATION-ATTACK-PHP/933150.yaml

```diff
@@ -567,3 +567,19 @@ tests:
         output:
           log:
             no_expect_ids: [933150]
+  - test_id: 34
+    desc: False positive with SprintForTheCause matching printf
+    stages:
+      - input:
+          dest_addr: 127.0.0.1
+          headers:
+            Host: localhost
+            User-Agent: "OWASP CRS test agent"
+            Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+          method: POST
+          port: 80
+          uri: "/post/?test=SprintForTheCause"
+          version: "HTTP/1.1"
+        output:
+          log:
+            no_expect_ids: [933150]
```

Comment:
Just for completitude, can we add another negative test into 933150 where we also don't match `printf`? With that it is ready to marge imho.